pkg/webui/styles/variables/generic.styl

```diff
@@ -70,6 +70,12 @@ $fs = {
   'xl4': 3.428rem // ~48px
 }
 
+$fsv2 = {
+  's': 0.75rem,
+  'm': 1rem,
+  'l': 1.25rem
+}
```

Comment:
Do we already have a full list of font sizes @pierrephz ?

Reply:
I think we said we stick with 3 font sizes